### ISSUE TITLE
Mythril dies on contract with non-ascii characters

### ISSUE THREAD

**Issue by alexaltair** [ConsenSys/mythril#244]
**Describe the bug**
I have a contract that happened to contain some non-ASCII characters in the comments. This compiles fine through solc, but when I ran `myth -x --solv 0.4.24 ./MyContract.sol` I got the following exception.
```
Traceback (most recent call last):
  File "/usr/local/bin/myth", line 4, in <module>
    __import__('pkg_resources').run_script('mythril==0.17.14', 'myth')
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 658, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1438, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python3.6/dist-packages/mythril-0.17.14-py3.6.egg/EGG-INFO/scripts/myth", line 9, in <module>
    mythril.interfaces.cli.main()
  File "/usr/local/lib/python3.6/dist-packages/mythril-0.17.14-py3.6.egg/mythril/interfaces/cli.py", line 163, in main
    address, _ = mythril.load_from_solidity(args.solidity_file)  # list of files
  File "/usr/local/lib/python3.6/dist-packages/mythril-0.17.14-py3.6.egg/mythril/mythril.py", line 286, in load_from_solidity
    contract = SolidityContract(file, contract_name, solc_args=self.solc_args)
  File "/usr/local/lib/python3.6/dist-packages/mythril-0.17.14-py3.6.egg/mythril/ether/soliditycontract.py", line 40, in __init__
    code = file.read()
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2223: ordinal not in range(128)
```
I removed the non-ASCII characters, re-ran the command, and it worked fine.

**Expected behavior**
I expected that, since solc can compile it, mythril would be able to analyze it in some way.

**Environment:**
 - OS: Ubuntu 16.04
 - Mythril commit 63366f7c0e9bc4877e99e8dd08a323f85d0563e1

**Additional context**
Technically what I did was clone the mythril repo, ran `docker build mythril`, and used that container to run further commands. But it also failed in the same way on CircleCI when following the install instructions.

**Comment by birdofpreyru** [comment]
@JoranHonig just to double-check, do I take it to Topcoder to fix?

**Comment by JoranHonig** [comment]
I think this was one of the challenges @b-mueller mentioned last Wednesday, that should be part of a topcoder challenge. @b-mueller should this still be part of a topcoder challenge?

**Comment by birdofpreyru** [comment]
Cannot find that card now. But, whatever, scheduled this a small stand-alone challenge: https://www.topcoder.com/challenges/30066271/

**Comment by birdofpreyru** [comment]
Fixed in **v0.18.4**
